src/components/Nav/Nav.js

```diff
@@ -1,7 +1,83 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+import * as S from './NavStyle';
+import { useNavigate } from 'react-router-dom';
 
 const Nav = () => {
-  return <div />;
+  const token = true;
+  //Todo
+  //token 생성시  추가 구현
+
+  const navigate = useNavigate();
+
+  const handleLogout = () => {
+    localStorage.removeItem('token');
+    alert('로그아웃 되었습니다.');
+  };
+  return (
+    <S.Header>
+      <S.Container>
+        <S.TitleLogo>
+          <Link to="/" className="LinkLogo">
+            <img
+              src="../images/comebackhome2.png"
```

Comment:
public/data의 이미지는 절대경로로 접근해야합니다.
지금은 상대경로로 접근하고있네요

```suggestion
              src="/images/comebackhome2.png"
```